glslc/src/file_includer.h

```diff
@@ -34,22 +36,36 @@ class FileIncluder : public shaderc::CompileOptions::IncluderInterface {
  public:
   FileIncluder(const shaderc_util::FileFinder* file_finder)
       : file_finder_(*file_finder) {}
-  shaderc_includer_response* GetInclude(const char* filename) override;
-  void ReleaseInclude(shaderc_includer_response* data) override;
+
+  // Resolves a requested source file of a given type from a requesting
+  // source into a shaderc_include_result whose contents will remain valid
+  // until it's released.
+  shaderc_include_result* GetInclude(const char* requested_source,
+                                     shaderc_include_type type,
+                                     const char* requesting_source,
+                                     size_t include_depth) override;
+  // Releases an include result.
+  void ReleaseInclude(shaderc_include_result* include_result) override;
+
+  // Returns a reference to the member storing the set of included files.
   const std::unordered_set<std::string>& file_path_trace() const {
```

Comment:
For another CL.  This methods should be called "included_files" 
It's a cheap accessor to the value of a member variable of the same name.